robojudge.ipynb

```diff
@@ -186,8 +189,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hist_file_name = \"validation/data/\" + hist_end_date.strftime('%Y-%m-%d') + \"_historical_ip.csv\"\n",
-    "hist_file_name"
+    "# hist_file_name = \"validation/data/\" + hist_end_date.strftime('%Y-%m-%d') + \"_historical_ip.csv\"\n",
```

Comment:
I'm not keen on commented-out code, even in Notebooks, as it sends mixed signals to the reader which can be confusing ("is this needed or not?")

Reply:
Removed - it's not the role of robojudge to generate this data (anymore). It's now done in the `scenario_generator` notebook.
We need to revisit and cleanup this notebook but it's not for this PR.